plugins/fallbacker/timer/timer_fallbacker.go

```diff
@@ -43,17 +45,25 @@ func (t *Fallbacker) Description() string {
 
 func (t *Fallbacker) DefaultConfig() string {
 	return `
+# The forwarder max retry times.
 max_times: 3
+# The latency_factor is the standard retry duration, and the time for each retry is expanded by 2 times until the number 
+# of retries reaches the maximum.(Time unit is millisecond.)
 latency_factor: 2000
+# The max retry latency time.(Time unit is millisecond.)
+max_latency_time: 5000
 `
 }
 
 func (t *Fallbacker) FallBack(batch event.BatchEvents, forward api.ForwardFunc) bool {
-	currentLatency := t.latencyFactor
-	for i := 1; i < t.maxTimes; i++ {
+	currentLatency := t.LatencyFactor
+	for i := 1; i < t.MaxTimes; i++ {
```

Comment:
```suggestion
	for i := 1; i <= t.MaxTimes; i++ {
```

Reply:
when trigger the fallbacker, already trigger once, so from 1 to add